README.md

```diff
@@ -1 +1,40 @@
-# equalizer
\ No newline at end of file
+# Wolox Equalizer
+[![circle-ci](https://img.shields.io/circleci/project/github/Wolox/wolox-equalizer.svg)](https://circleci.com/gh/Wolox/wolox-equalizer)
+[![npm](https://img.shields.io/npm/v/wolox-equalizer.svg)](https://www.npmjs.com/package/wolox-equalizer)
+
+[![FEArmy](./assets/FEA_icon.png)](https://github.com/orgs/Wolox/teams/front-end-army/members)
+## Features
+
+- Reset the default styles of the browser.
+- Contains our Wolox standards for normalizing front-end projects.
+
+## Installation
+
+#### npm
+```bash
+npm i wolox-equalizer
+```
+
+#### yarn
+```bash
+yarn add wolox-equalizer
+```
+
+## Usage
+
+1- Include the equalizer.scss before the index file of your CSS/SCSS.
+
+```js
```

Comment:
can we add a scss way, I think `@import 'wolox-equalizer/equalizer.scss'` in the application.scss would be a bit tidier

Reply:
```js
      {
         loader: 'sass-loader',
        options: {
          includePaths: ['node_modules']
        }
    }
```